op-chain-ops/genesis/genesis.go

```diff
@@ -54,6 +54,7 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		TerminalTotalDifficulty:       big.NewInt(0),
 		TerminalTotalDifficultyPassed: true,
 		BedrockBlock:                  new(big.Int).SetUint64(uint64(config.L2GenesisBlockNumber)),
+		RegolithTime:                  config.RegolithTime(block.Time()),
```

Comment:
Will also need to set this in `op-chain-ops/genesis/db_migration.go` as part of the hardfork configs starting on line 283. I'm tracking this for myself in CLI-3384, but should be as easy as a one-line change in this PR.

Reply:
Yep makes sense. Done.